app/src/test/java/ch/hikemate/app/model/facilities/FacilitiesRepositoryOverpassTest.kt

```diff
@@ -110,34 +106,6 @@ class FacilitiesRepositoryOverpassTest {
     `when`(client.newCall(any())).thenReturn(mockCall)
   }
 
-  @OptIn(ExperimentalCoroutinesApi::class)
-  @Test
-  fun testGetFacilities_integrationTest() = runTest {
-    val realFacilitiesRepository = FacilitiesRepositoryOverpass(OkHttpClient())
-
-    var onSuccessCalled = false
-    var routesSize = 0
-
-    withContext(Dispatchers.Default.limitedParallelism(1)) {
-      withTimeout(5.seconds.toJavaDuration()) { // 5 seconds timeout
-        // Convert callback-based API to suspending function
-        suspendCancellableCoroutine { continuation ->
-          realFacilitiesRepository.getFacilities(
-              bounds = testBoundsNormal,
-              onSuccess = { routes ->
-                routesSize = routes.size
-                onSuccessCalled = true
-                continuation.resume(Unit, {})
-              },
-              onFailure = { error -> continuation.resumeWithException(error) })
-        }
-      }
-    }
-
-    assertTrue(onSuccessCalled)
-    assertNotEquals(0, routesSize)
-  }
-
```

Comment:
Question: Why was this test removed?

Reply:
Good question: I removed it as it is not a unit test and therefore should not be here.

Reply:
Wasn't there somewhere else you could've put it instead of removing it? Or do we assume that this is going to be tested by a larger E2E test later on?

Reply:
I will be tested by E2E 3 I guess.

Reply:
Okay, that makes sense!